misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/utils/BindgenConstants.java

```diff
@@ -49,9 +49,12 @@ private BindgenConstants() {
     public static final String CONSTANTS_FILE_NAME = "Constants.bal";
     public static final String ARRAY_UTILS_FILE_NAME = "ArrayUtils.bal";
     public static final String DEPENDENCIES_DIR = "dependencies";
-    public static final String ERROR_TYPES_DIR = "error_types";
+    static final String TARGET_DIR = "target";
```

Comment:
To fix the merge conflict issue.
```suggestion
    public static final String ERROR_TYPES_DIR = "error_types";
    static final String TARGET_DIR = "target";
```